packages/backend/src/open_payments/auth/grant.ts

```diff
@@ -104,7 +102,7 @@ export class Grant {
 
   public toJSON(): GrantJSON {
     return {
-      active: this.active,
+      active: true,
```

Comment:
So because an inactive grant can't be created, this will always be true?

Reply:
correct